scripts/ci/pr-bot/processNewPrs.ts

```diff
@@ -45,15 +45,24 @@ import { CheckStatus } from "./shared/checks";
  * (in which case that's all we need to do).
  */
 function needsProcessed(pull: any, prState: typeof Pr): boolean {
-  if (!pull.labels.find((label) => label.name.toLowerCase() === "go")) {
+  if (
+    !pull.labels.find(
+      (label) =>
+        label.name.toLowerCase() === "go" ||
+        label.name.toLowerCase() == "python"
+    )
+  ) {
     console.log(
-      `Skipping PR ${pull.number} because it doesn't contain the go label`
+      `Skipping PR ${pull.number} because it doesn't contain the go or python labels`
     );
     return false;
   }
-  const firstPrToProcess = new Date(2022, 2, 2, 20);
+  const firstPrToProcess = new Date(2022, 7, 16, 14); // June 16 2022, 14:00 UTC (note that Java months are 0 indexed)
```

Comment:
did you mean to use (2022, 5, 16, 14) if they are 0-indexed?

Reply:
Good catch - fixed